src/hooks/usePatientData.tsx

```diff
@@ -14,19 +14,21 @@ function calculateAge(birthDate: Date): number {
   }
 }
 
-const patientGenderMap = {
-  female: 'F',
-  male: 'M',
-  other: 'O',
-  unknown: 'U',
-};
```

Comment:
This variable doesn't seem to be used anywhere else, so I've moved it into the `usePatientData` hook body.